scripts/Costs.py

```diff
@@ -332,7 +333,7 @@ def tradeCosts(costType, regions, years):
     return dfOut
 
 def getUsaCapitalCost():
-    # PURPOSE: Creates capitalCost file from USA data
+    # PURPOSE: Creates capitalCosts from USA data (excluding transmission)

```

Comment:
Why excluding transmission? Isn't Capital Costs -for- transmission?

Reply:
USA transmission costs are populated using the function `def tradeCosts()` now